docs/source-1.0/guides/building-models/build-config.rst

```diff
@@ -298,6 +298,49 @@ Only the following shape type changes are supported:
         }
 
 
+.. _excludeShapesBySelector-transform:
```

Comment:
Missing the port to 2.0 docs.

Reply:
🤦 I'll fix that, it should only be for the 2.0 docs.

Reply:
Updated, thanks!